Scroll to top button (#665)

varunwho opened the pull request:
### Scroll to top button

https://user-images.githubusercontent.com/53107832/122255536-d8828180-ceeb-11eb-8b3b-5608f0b37c58.mp4

smaranjitghose commented on assets/js/dynamicTitle.js:
```diff
@@ -1,5 +1,3 @@
-
```
Avoid these changes for now

smaranjitghose commented on assets/js/app.js:
```diff
@@ -213,13 +213,18 @@ let footer = $(`
 </footer>
 `);
 
-
```
Add appropriate comments

smaranjitghose commented:
Reduce the size of the button by 25% of it's present size. The color of the button should be different while on the footer and on the remainder of the page

varunwho commented:
> Reduce the size of the button by 25% of it's present size. The color of the button should be different while on the footer and on the remainder of the page

Yup I will do it

anushbhatia reviewed (changes requested):
On selection of the button avoid that black box around it if possible.

varunwho commented:
#665 
### **I've done these changes**
- Reduced the size
- Added the microanimation which was suggested
- Removed the outline which appeared after the click
- The color of the button changes when it hits the footer

smaranjitghose commented:
@varunwho This looks perfect. Just incorporate the recent changes to the project to remove the merge conflict.

anushbhatia reviewed (changes requested):
When we move to the top of the page it should vanish automatically.

varunwho commented:
Ignore the latest commit here, I've made a new PR
